tests/unit_tests/experiment_plans/test_grid_detect_then_xray_centre_plan.py

```diff
@@ -26,11 +28,10 @@ def _fake_grid_detection(
     snapshot_dir: str,
     grid_width_microns: float = 0,
     box_size_um: float = 0.0,
+    smargon=None,
```

Comment:
I think https://github.com/DiamondLightSource/hyperion/pull/1423/commits/2274ac616c1bacec1df04d315ef67f4e36e94556 is slightly cleaner, feel free to disagree though